package.json

```diff
@@ -216,7 +216,21 @@
           }
         }
       }
-    }
+    },
+    "languages": [
+      {
+        "id": "ignore",
+        "extensions": [
+          ".remarkignore"
+        ]
+      },
+      {
+        "id": "json",
+        "filenames": [
+          ".remarkrc"
+        ]
+      }
+    ]
```

Comment:
This is new-ish to me.
Does this only impact people cloning this repo?
Or does this somehow tell vscode how to highlight these files if `remark` is installed?

Reply:
This is part of the plugin manifest, not local development settings.

> Or does this somehow tell vscode how to highlight these files if remark is installed?

:point_up: This exactly

I basically copied it from https://github.com/microsoft/vscode-eslint/blob/main/package.json#L490-L503

Reply:
Interpreting the `:+1:` by Christian as an ✅ review